test/sql/pg_dump_unprivileged.sql

```diff
@@ -13,6 +13,13 @@ CREATE USER dump_unprivileged CREATEDB;
 \c template1 dump_unprivileged
 CREATE database dump_unprivileged;
 
+-- pg_dump will fail due to job_errors permission missing
+-- https://github.com/timescale/timescaledb/issues/5449
+-- as a workaround GRANT the job_errors privilege to the user
+\c dump_unprivileged :ROLE_SUPERUSER
+GRANT SELECT ON TABLE _timescaledb_internal.job_errors TO dump_unprivileged;
+
+\c template1 dump_unprivileged
```

Comment:
Shouldn't this be part of a separated PR?

Reply:
Removed this as well